main.py

```diff
@@ -1,8 +1,93 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Request, Response
+from pydantic import BaseModel
+from typing import List
+import uvicorn
+import httpx
+import os
 
+# This is page access token that you get from facebook developer console.
+PAGE_ACCESS_TOKEN = 'EAAGQoF3oHhEBAFIZC1ZCx7wx4ngCcgZAKwEZCtBzTQkGOQXu4k6jENLH35bV0F3hWBMO8Jlasin9BsvgU206541dnRePHXXz5TZBBXEXYfkxuZBuZAntS8nJ4VMFu1RgZC9AiwMCKN69J9R5sZAgmttIPyAZAYrgddZCP9mZBsM9NZAxskuFUgB5p5Tm9'
+VERIFY_TOKEN = 'seattlebot'
```

Comment:
normally we store the `TOKEN` in .`env` file, and load into environment using `dotenv` package, https://pypi.org/project/python-dotenv/